sakai-evaluation-api/src/java/org/sakaiproject/evaluation/beans/EvalBeanUtils.java

```diff
@@ -331,9 +340,40 @@ public void setEvaluationDefaults(EvalEvaluation eval, String evaluationType) {
         	}
         }
         
-        if (eval.getResultsSharing() == null) {
-            eval.setResultsSharing( EvalConstants.SHARING_VISIBLE );
+        // Section awareness default controlled by sakai.property
+        if( eval.getSectionAwareness() == null )
+        {
+        	if( EVALSYS_SECTION_AWARE_DEFAULT )
+        		eval.setSectionAwareness( Boolean.TRUE );
+        	else
+        		eval.setSectionAwareness( Boolean.FALSE );
```

Comment:
Can cut this down to:
eval.setSectionAwareness(Boolean.valueOf(EVAL_SECTION_AWARE_DEFAULT));